lib/jefferies_tube.rb

```diff
@@ -21,7 +21,7 @@ class Configuration
     def initialize
       if defined?(Rails)
         @environment = ::Rails.env.downcase || nil
-        @prompt_name = ::Rails.application.class.parent_name || nil
+        @prompt_name = ::Rails.application.class.module_parent_name || nil
```

Comment:
I think I should do something similar to this to keep support Rails <6: https://github.com/igorkasyanchuk/any_login/pull/24/files